### PR TITLE
feat(permit): add permit info json schema

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Part of #293

Adding json schema for permit info

# Test

Unit tests

### CODE REVIEW

**Comment by alfetopito** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/permitInfo/permitInfo.schema.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+              "type": "string",
+              "description": "Optional version number, as a string",
+              "pattern": "^\\d+$"
+            },
+            "type": {
+              "type": "string",
+              "description": "Type of permit",
+              "enum": [
+                "eip-2612",
+                "dai-like"
```
Comment (anxolin) [inline src/permitInfo/permitInfo.schema.json:25]: nit: if we want a more homogeneous definition, we can add an option here for `unsupported` so we don't need the `oneOff` which can give problems or is harder interface.

 I should have said this in permit PRs, and also in the issue where you mentioned the schema. Anyways, is not too important, and yours is of course slightly more compact, so feel free to disregard the comment

**Review by anxolin — APPROVED** [review]
Great tests!

File: src/permitInfo/permitInfo.schema.json (hunk 1 above)
Comment (alfetopito) [reply src/permitInfo/permitInfo.schema.json:25]: I like the suggestion, actually.
It's just that it creates some work for the CoW Swap implementation.
I'll create a task as a good to have and address it if there's time.

Comment (alfetopito) [reply src/permitInfo/permitInfo.schema.json:25]: https://github.com/cowprotocol/token-lists/issues/295

Comment (anxolin) [reply src/permitInfo/permitInfo.schema.json:25]: yes, not an issue. thx for tracking the suggestion anyways

**Pull request merged**